frontend/src/features/documents/list/DocumentListView.test.tsx

```diff
@@ -13,6 +14,17 @@ const storeState = {
   [lookupCodesSlice.name]: { lookupCodes: mockLookups },
 };
 
+jest.mock('@react-keycloak/web');
+(useKeycloak as jest.Mock).mockReturnValue({
+  keycloak: {
+    userInfo: {
+      organizations: [1],
+      roles: [],
+    },
+    subject: 'test',
+  },
+});
+
```

Comment:
nit: we don't need to do this anymore. The **render** function has this built-in if you pass the option useMockAuthentication: true

```tsx

const component = render(<Blah />, { ...renderOptions, store: storeState, useMockAuthentication: true});

```

Reply:
FYI - you DO need to keep `jest.mock('@react-keycloak/web');`